app/build.gradle

```diff
@@ -204,6 +204,10 @@ dependencies {
     implementation 'com.android.support:palette-v7:27.1.1'
     implementation project(':openCVLibrary330')
 
+    /* for devMethods.thirdparty.pokebattler, temporally set for All Builds */
+    implementation 'org.json:json:20160810' //used to generate moveset list
```

Comment:
I needed to comment this to actually be able to build the "offlineRelease" build. It mentioned that Android comes with it built in. It also uses GSON, so maybe use that instead?

Reply:
I have no idea whats going on here, so I guess do whatever you need to make it work. This is all old stuff from the original PR, and I've not really gotten a grasp on Gradle yet.

Are you recommending we use gson or switch to json? I'm having a bit of trouble parsing your comment 😅

Reply:
I wanted to say, that this project already uses GSON which is also a JSON parser. So I'm wondering why it is also requiring JSON. Maybe I'm wrong and they don't do the same.

Additionally it didn't compile for me because it was a duplicated requirement. I'll try to post the exact message later.